log/field.go

```diff
@@ -20,13 +23,13 @@ const (
 )
 
 // Field instances are constructed via LogBool, LogString, and so on.
-// Tracing implementations may then handle them via the Field.Process
+// Tracing implementations may then handle them via the Field.Marshal
 // method.
 //
 // "heavily influenced by" (i.e., partially stolen from)
 // https://github.com/uber-go/zap
 type Field struct {
-	key          string
+	Key          string
```

Comment:
would it make more sense to expose `Key()` instead? That would keep the `Field` immutable.